### ISSUE TITLE
Unique / Novel scores

### ISSUE THREAD

**Issue by j-min** [jamespark3922/visual-comet#4]
Hi @jamespark3922, I'm running experiments with visualcomet and comparing with your baseline in the paper.

Interestingly, my training run always have 'Unique' scores higher than 'Novel' scores, which is opposite in Table 2 in your paper. I'm using your evaluation script.
Actually, 0.00 Unique scores of 'No Input' row doesn't make sense to me. You're generating 5 sentences per input. Then isn't the lower bound of 'Unique' score 20%?
Maybe I'm wrong. Could you please double check whether the values of the Unique and Novel columns are not accidentally swapped? 
![image](https://user-images.githubusercontent.com/18069263/103248304-138dfa00-4938-11eb-8526-215e75a70c99.png)

**Comment by jamespark3922** [comment]
In the paper, instead of calculating the score over 5 sentences for each image & event pair, I calculated them over all the sentences in the validation set (i.e. treat each inference sentences as individual sentence and get the diversity over all the inference sentences). 

Because we are using sampling in inference and generating almost different sentences for each image + event pair, I found that calculating unique within 5 sentences is trivial and gets a high number.
